Project.toml

```
@@ -70,5 +70,5 @@ SciMLBase = "2.68"
 StaticArrays = "1.9"
 StatsBase = "0.34"
 SurfaceFluxes = "0.12.1"
-Thermodynamics = "0.13.1"
+Thermodynamics = "0.13.1, 0.14"
```

```suggestion
Thermodynamics = "0.14"
```
(0.14 is required for ClimaParams 0.12)